config/default.json

```diff
@@ -4,7 +4,34 @@
     "referrerPolicy": "origin",
     "hpkp": {
```

Comment:
hpkp is deprecated and shouldn't be used in new projects

Reply:
Thanks @timja , I don't know too much about it. I have read about it just now here - https://ordina-jworks.github.io/security/2018/02/12/HPKP-deprecated-what-now.html . Should i do this - Expect-CT?

express-template is still using hpkp, so that has to be changed too.

Reply:
code has been removed by @adusumillipraveen in expressjs-template which consumes this, he missed the config and is going to remove it now 😉

Reply:
applied express-template changes to this repo